src/site/stages/build/process-cms-exports/schemas/input/node-press_release.js

```diff
@@ -3,7 +3,15 @@
 module.exports = {
   type: 'object',
   properties: {
+    uid: {
+      type: 'array',
+      maxItems: 1,
+      items: { $ref: 'EntityReference' },
+    },
     title: { $ref: 'GenericNestedString' },
+    created: { $ref: 'GenericNestedString' },
+    changed: { $ref: 'GenericNestedString' },
```

Comment:
Looking at the GraphQL output, I didn't see the `changed` field. Do we still need to include it?

Reply:
I added just for consistency since we have been doing it in our previous transformers